apps/nextra/pages/en/build/move/book/functions.mdx

```diff
@@ -24,24 +24,25 @@ Module functions, by default, can only be called within the same module. These i
 
 ```move
 address 0x42 {
-  module m {
+module m {
     fun foo(): u64 { 0 }
+
     fun calls_foo(): u64 { foo() } // valid
-  }
+}
 
-  module other {
+module other {
     fun calls_m_foo(): u64 {
-      0x42::m::foo() // ERROR!
-//    ^^^^^^^^^^^^ 'foo' is internal to '0x42::m'
+        0x42::m::foo() // ERROR!
+        //    ^^^^^^^^^^^^ 'foo' is internal to '0x42::m'
```

Comment:
The `^^^^` seems to be misaligned with this change. There are several instances of these in the file, better to grep and fix all of them.

Reply:
I think I fixed all instances. @gregnazario could that have come from some autoformatting you applied to the other example?

Reply:
I did them all manually, for some reason the prettier formatter destroys MDX files (we probably have to play with the settings)